store/src/java/com/zimbra/cs/imap/ImapHandler.java

```diff
@@ -2475,7 +2488,12 @@ boolean doSTATUS(String tag, ImapPath path, byte status) throws ImapException, I
                 sendNO(tag, "STATUS failed");
                 return true;
             }
-
+            if(credentials != null) {
+                MailboxStore mbox = credentials.getMailbox();
+                if(mbox != null) {
+                    mbox.noOp();
```

Comment:
Given that doNOOP uses try/catch and canContinue round this, is it worth having that here?